high_card.rb

```diff
@@ -1,5 +1,32 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+require "pry"
+
+ranks = [ 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+players =[
+  
+]
+
+deck = ranks.product(suits).each do |card| "#{card[0]} #{card[1]}"
+end
+
+playing_deck = deck.shuffle
+
+def enter_players
+  puts "please enter player name"
+    answer = gets.chomp
+    if answer == "play"
+      run_game
+    else players << answer
+end
+
+def run_game (players)
+  players.each do |players|
+    puts " #{[:name]} draws playing_deck[0]"
+
+end
```

Comment:
This will error out because you need two `end` statements -- one for the method and one for the the `.each`